docs/README.md

```diff
@@ -1,28 +1,62 @@
 # Crossplane
 
-Crossplane is an open source multicloud control plane. It introduces workload and resource abstractions on-top of existing managed services that enables a high degree of workload portability across cloud providers. A single crossplane enables the provisioning and full-lifecycle management of services and infrastructure across a wide range of providers, offerings, vendors, regions, and clusters. Crossplane offers a universal API for cloud computing, a workload scheduler, and a set of smart controllers that can automate work across clouds.
+Crossplane is an open source multicloud control plane to manage your cloud-native applications and infrastructure across environments, clusters, regions and clouds. It enables deployment of applications and managed services from `kubectl` with your choice of cloud or on-premise providers, including independent service offerings. 
 
-<h4 align="center"><img src="media/arch.png" alt="Crossplane" height="400"></h4>
+Crossplane can be installed into an existing Kubernetes cluster to add managed service provisioning or deployed as a dedicated control plane for multi-cluster management and workload scheduling.
 
-Crossplane presents a declarative management style API that covers a wide range of portable abstractions including databases, message queues, buckets, data pipelines, serverless, clusters, and many more coming. It’s based on the declarative resource model of the popular [Kubernetes](https://github.com/kubernetes/kubernetes) project, and applies many of the lessons learned in container orchestration to multicloud workload and resource orchestration.
+Crossplane enables the community to build and publish Stacks to add more clouds and cloud services to Crossplane with support for out-of-tree extensibility and independent release schedules. 
```

Comment:
Did we already consider defining "out-of-tree extensibility" or linking to something that does? I am thinking we would want to do that, because this is not a term I was familiar with before I heard it a lot in the context of Crossplane. And maybe some of our readers will also not be familiar with it.

Reply:
good idea, it a popular term in Kubernetes: https://kubernetes.io/docs/search/?q=out-of-tree

will add it to the concepts / glossary